### PR TITLE
Separate dashboards into own ConfigMaps

### PR DESCRIPTION
**Pull request by metalmatze**
Tested in my cluster.
In a future PR I'd like to enable those dashboards to be put in separate folders.

### CODE REVIEW

File: examples/custom-ini.jsonnet [hunk 1]
```diff
@@ -15,14 +15,15 @@ local grafana = ((import 'grafana/grafana.libsonnet') + {
...
-  grafana.dashboardSources,
-  grafana.dashboardDatasources,
-  grafana.deployment,
-  grafana.serviceAccount,
-  grafana.service +
-  service.mixin.spec.withPorts(servicePort.newNamed('http', 3000, 'http') + servicePort.withNodePort(30910)) +
-  service.mixin.spec.withType('NodePort'),
-])
+k.core.v1.list.new(
+  [dashboard for dashboard in grafana.dashboardDefinitions] + [
```
Comment (brancz) [inline examples/custom-ini.jsonnet]: `grafana.dashboardDefinitions` is already an array, it should be possible to just concatenate the two arrays without the `for`.

**Comment by brancz** [comment]
Generally looks good, I think the example code can be even a bit simpler as described in the comment. Also if you don't mind, I'd like to prepare a kube-prometheus patch integrating this, before I merge this.

**Comment by metalmatze** [comment]
Yes. Sure. Test it thoroughly and then we can merge. I'm copying the changes to my `/vendor` right now anyway. So I can wait a bit.

**Comment by metalmatze** [comment]
Updated to merge dashboardDefinitions as array directly. :relaxed:

**Comment by brancz** [comment]
Perfect, thank you!

**Pull request merged**